src/time_travel/patchers/tests/test_select.py

```diff
@@ -69,19 +71,19 @@ def test_multiple_fds_for_same_time(self):
         fd4 = mock.MagicMock(name='fd4')
         unwaited_fd = mock.MagicMock(name='unwaited_fd')
         
-        self.events_pool.add_future_event(3,
+        self.events_pool.add_future_event(100003,
```

Comment:
I really don't like how all those tests look.
I prefer you add a constant for "zero_time" and write `ZERO_TIME + 3` or somthing like it in all tests.

Reply:
Done as discussed.